### ISSUE TITLE
Record query

### ISSUE THREAD

**Issue by davidnicholls125** [johnvanbreda/naturespot-d8#56]
An old record, 8324, has just appeared in the verification queue (as + new species) with no details in the comments tab. It was last updated yesterday but no-one seems to know anything about it. Can you check who edited it please?
You mentioned recently that you could set up a way for us to see the full log details of a record. I can't remember how log you thought it would take but if it is just an hour or two then go ahead please.
![image](https://user-images.githubusercontent.com/80914584/129860345-fd23fb9f-5ae6-42bd-94f4-638e33f29f78.png)

**Comment by johnvanbreda** [comment]
There were a couple of records, 8324 and 8325, which pointed to a deleted copy of the taxon name Larch Ladybird. This name must have been a very early addition to the system as it had a blank TVK. When I ran some cleanup code on Tuesday, it detected these and mapped them to the other name that has a blank TVK, i.e. + new species. 

I ran a query to check these were the only examples. I also repointed these 2 records back to the current correct Larch Ladybird name entry.
